### DISCUSSION TITLE
Valkey support

### DISCUSSION THREAD

**Discussion by atalasuibm** [redisson/redisson#5870] [Q&A]
We are looking into replacing Redis with Valkey in our product. Can Redisson can be used connect to Valkey? The Readme states that Redisson is Valkey compatible. Are there any examples showing how to connect to Valkey?

**Comment by mrniko** [comment]
Redisson is fully compatible with Valkey. You can use the same settings and url format as you used for Redis

**Reply by atalasuibm** [reply]
Thanks!
